CriticalMass/Theme.swift

```diff
@@ -8,7 +8,7 @@
 
 import UIKit
 
-enum Theme: Int {
+enum Theme: String {
```

Comment:
This change means that users would loose their current preference right?

Reply:
Oh. Not sure 🤔 
But I think so yes. Since it wants a string from the defaults it should return nil and then be a light theme by default on <iOS 13

Reply:
We should probably revert that change to prevent data loss

Reply:
Since it is just a theme change and on older OS I would merge it since it makes the API clearer imo

Reply:
Yes it makes it clearer but in my opinion the benefits are not strong enough to accept loosing the preference but I’m also okay with merging it